### PR TITLE
[DEVX-153]:  Model Predict Examples

### PR DESCRIPTION
**Pull request by sainivedh**
## What
 - Add Model Predict examples `model_predict.ipynb`
 - Covers all input types (image, text, audio, video)
 - Customising prediction parameters(concepts, threshold..etc)
 - Listing of models

### CODE REVIEW

**Review by ackizilkale — APPROVED** [review]
Looks good!

File: models/model_predict.ipynb [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.10"
+  },
+  "orig_nbformat": 4
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:580]: In the last cell , we can add `Clarifai Resources` like in basics and dataset_upload notebooks..

File: models/model_predict.ipynb [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_url = \"https://clarifai.com/salesforce/blip/models/general-english-image-caption-blip\"\n",
+    "image_url = \"https://s3.amazonaws.com/samples.clarifai.com/featured-models/image-captioning-statue-of-liberty.jpeg\"\n",
+    "model_prediction = Model(model_url).predict_by_url(image_url, input_type=\"image\")\n",
+    "print(model_prediction)"
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:334]: Same here, we can have like this `print(model_prediction.outputs[0].data.text.raw)`

File: models/model_predict.ipynb [hunk 3]
```diff
@@ -0,0 +1,580 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_url = \"https://clarifai.com/eleven-labs/audio-generation/models/speech-synthesis\"\n",
+    "\n",
+    "model_prediction = Model(model_url).predict_by_bytes(b\"Hello, How are you doing today!\", \"text\")\n",
+    "print(model_prediction.outputs[0].data.audio.base64)"
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:304]: We should save this audio or play it like we did for image. WDYT?

File: models/model_predict.ipynb [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Asking Claude2 to write a tweet on future of AI\n",
+    "model_url = \"https://clarifai.com/anthropic/completion/models/claude-v2\"\n",
+    "model_prediction = Model(model_url).predict_by_bytes(b\"Write a tweet on future of AI\", input_type=\"text\")\n",
+    "print(model_prediction)\n"
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:213]: For maintaining convention, we can have the completion alone here, like below `print(model_prediction.outputs[0].data.text.raw)`

File: models/model_predict.ipynb [hunk 5]
```diff
@@ -0,0 +1,580 @@
...
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import Model class\n",
+    "from clarifai.client.model import Model"
+   ]
+  },
+  {
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:93]: Maybe add `For colab` cell to get the files in this repo in Colab..

File: models/model_predict.ipynb [hunk 6]
```diff
@@ -0,0 +1,580 @@
...
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Installation\n",
+    "\n",
+    "It is required to set `CLARIFAI_PAT` as Environment Variable.\n",
+    "\n",
+    "More info on [Personal Access Tokens](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens)"
```
Comment (sanjaychelliah) [inline models/model_predict.ipynb:61]: Minor Suggestion, All the notebooks following the same structure in PAT intro will good. WDYT?

**Review by sanjaychelliah — APPROVED** [review]
LGTM

**Pull request merged**